internal/pkg/cli/svc_init.go

```diff
@@ -75,6 +80,8 @@ type initWkldVars struct {
 	name           string
 	dockerfilePath string
 	image          string
+	subscribeTags  []string
```

Comment:
nit: When I saw tags I thought of AWS tags. What do you think of this:
```suggestion
	subscriptions  []string // or serializedSubscriptions
```